packages/editor-ui/src/components/NodeWebhooks.vue

```diff
@@ -94,7 +94,7 @@ export default defineComponent({
 			}
 
 			return (this.nodeType as INodeTypeDescription).webhooks!.filter(
-				(webhookData) => webhookData.restartWebhook !== true,
+				(webhookData) => webhookData.restartWebhook !== true && !webhookData.hasLifecycleMethods,
```

Comment:
@elsmr just FYI: we've reverted this check in https://github.com/n8n-io/n8n/pull/7783

For more context: [Slack thread](https://n8nio.slack.com/archives/C035KBDA917/p1700666961614479).